site/_data/docs/privacy-sandbox/toc.yml

```diff
@@ -93,8 +93,12 @@
           title: i18n.docs.privacy-sandbox.experiment-participate
         - url: /docs/privacy-sandbox/attribution-reporting/chrome-shipping
           title: i18n.docs.privacy-sandbox.attribution-reporting-chrome-shipping
-        - url: /docs/privacy-sandbox/attribution-reporting-debugging
-          title: i18n.docs.privacy-sandbox.attribution-reporting-debugging
+        - title: i18n.docs.privacy-sandbox.attribution-reporting-debugging-resources
+          sections:
+            - url: docs/privacy-sandbox/attribution-reporting-debugging
```

Comment:
```suggestion
```

As we're adding all articles to the navigation, I don't believe the top page needs to be included. There isn't so much additional context.